Sources/API/Debug.swift

```diff
@@ -1,51 +1,60 @@
 public class Debug {
+    /// Default format option
     public static let defaultOption = Option(indent: 4)
```

Comment:
Should I make to `private` ...?

Reply:
I think so.

Reply:
I've been on the fence but I leave it as is.